src/web/mjs/connectors/MangaDisk.mjs

```diff
@@ -0,0 +1,37 @@
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
+export default class MangaDisk extends WordPressMangastream {
+
+    constructor() {
+        super();
+        super.id = 'mangadisk';
+        super.label = 'Manga Disk';
+        this.tags = [ 'manga', 'indonesian' ];
+        this.url = 'https://mangadisk.web.id';
+        this.path = '/manga/list-mode/';
+        this.queryPages = 'div#readerarea canvas';
+    }
+
+    async _getPages(chapter) {
+        const uri = new URL(chapter.id, this.url);
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, this.queryPages);
+
+        return data.map(hash => {
+            const payload = {
+                url: 'https://img.nesia.my.id/image?id='+hash.attributes["file-id"].value,
+                referer: this.url
+            };
+            return this.createConnectorURI(payload);
+        });
+    }
+
+    async _handleConnectorURI(payload) {
```

Comment:
If `this.url` is sufficient as `x-referer`, then there is no need to overwrite this method, the base class implementation will cover this when the connector adds the `x-referer` to the `requestOptions` in the constructor,
e.g.
```javascript
this.url = 'https://manga.adonisfansub.com';
this.requestOptions.headers.set('x-referer', this.url);
```